packages/wallets/src/features/cfd/screens/TradingPlatformStatus/TradingPlatformStatus.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
```

Comment:
For translations in wallets please use `Localize` or `useTranslations` from @deriv-com/translations instead as we're removing the old react-i18next implementation